[2025-06-09T06:14:35.130+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-09T06:14:35.187+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-09T06:14:33.299479+00:00 [queued]>
[2025-06-09T06:14:35.218+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-09T06:14:33.299479+00:00 [queued]>
[2025-06-09T06:14:35.219+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-09T06:14:35.272+0000] {taskinstance.py:2890} INFO - Executing <Task(_BranchPythonDecoratedOperator): check_file_exists_branch> on 2025-06-09 06:14:33.299479+00:00
[2025-06-09T06:14:35.294+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3917) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-09T06:14:35.300+0000] {standard_task_runner.py:72} INFO - Started process 3919 to run task
[2025-06-09T06:14:35.302+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_ocr_v2', 'check_file_exists_branch', 'manual__2025-06-09T06:14:33.299479+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmps_irj3h3']
[2025-06-09T06:14:35.311+0000] {standard_task_runner.py:105} INFO - Job 129: Subtask check_file_exists_branch
[2025-06-09T06:14:35.473+0000] {task_command.py:467} INFO - Running <TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-09T06:14:33.299479+00:00 [running]> on host 85194d6fbd18
[2025-06-09T06:14:35.806+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_ocr_v2' AIRFLOW_CTX_TASK_ID='check_file_exists_branch' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T06:14:33.299479+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T06:14:33.299479+00:00'
[2025-06-09T06:14:35.811+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-09T06:14:35.826+0000] {python.py:240} INFO - Done. Returned value was: get_file_info_list_task
[2025-06-09T06:14:35.828+0000] {branch.py:38} INFO - Branch into get_file_info_list_task
[2025-06-09T06:14:35.830+0000] {skipmixin.py:233} INFO - Following branch ('get_file_info_list_task',)
[2025-06-09T06:14:35.864+0000] {skipmixin.py:281} INFO - Skipping tasks [('end_task', -1)]
[2025-06-09T06:14:35.974+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-09T06:14:35.977+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_ocr_v2, task_id=check_file_exists_branch, run_id=manual__2025-06-09T06:14:33.299479+00:00, execution_date=20250609T061433, start_date=20250609T061435, end_date=20250609T061435
[2025-06-09T06:14:36.105+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-09T06:14:36.241+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-09T06:14:36.247+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
